scripts/create-rhsso-operator-index.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
```

Comment:
Would it not be better to re-use 3scale script to generate indexes but make it generic enough to support both?

Reply:
Jira opened for discussion https://issues.redhat.com/browse/MGDAPI-4481 cc @MStokluska @briangallagher